CSharp/ArticleProjects/ApplicationInsights/Microsfot.Azure.Batch.Samples.TelemetryStartTask/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.ApplicationInsights.Extensibility;
```

Comment:
It's actually mixed... I updated to put them all inside the namespace for this project though.